types/jest-cli/jest-tests.ts

```diff
@@ -0,0 +1,3 @@
+import jest from "jest-cli";
```

Comment:
You probably meant `import * as jest from "jest-cli";`.

Reply:
@andy-ms is that the proper import functionality?

Reply:
Right. `import jest from "jest-cli";` is a default import -- it gets the module's export named `default`. In this case there's no default export. `import * as jest from "jest-cli";` gets an object containing all exports.